environments/.env.latest

```diff
@@ -39,7 +39,10 @@ BUCKET_NAME=s3://sourcify-staging
 
 # IPFS config
 IPFS_SECRET=xxx
-IPFS_EXTERNAL_PORT=5050
+IPFS_EXTERNAL_PORT_8080=5050
+IPFS_EXTERNAL_PORT_4001=4001
+IPFS_EXTERNAL_PORT_4002_UDP=4002
```

Comment:
This one has no purpose IMO.

Reply:
Then I'll remove all it from the .env files and the mapping from ipfs.yaml.